### ISSUE TITLE
New Feature / Method

### ISSUE THREAD

**Issue by Lagicrus** [BlobEmoji/coindrop#8]
Instead of just responding to a typed message which can be a disadvantage to slow typers or mobile users.
Have alternate ways of responding to the "A coin has dropped message".

I was thinking that this could perhaps be via a reaction which is much easier and quicker to do.
By this I mean the bot reacts to itself to get around the issue of having reactions blocked for non-role users.
This can then be evolved to slightly combat selfbots by having say 3 reactions and the message says which to choose.

**Comment by Lagicrus** [comment]
@zebrawii ,  Basically instead of only just responding with ".pick" and the like, you can also react with a certain emote which is easier for mobile users. If that is what you were confused about

**Comment by Zeboto** [comment]
That seems almost too easy though. And it will be based on who's internet speed is the fastest again.

**Comment by Lagicrus** [comment]
True, it would be easy but you could make it harder for them having to choose one or the like
In terms of speed, isn't it technically already down to speed?
